File: Tatsuki_Yabe/Q8/Go/gauss.go
package main

func missingNumber2(nums []int) int {
	len := len(nums)
	intendedSum := len * (len + 1) / 2
	actualSum := 0

	for _, num := range nums {
		actualSum += num
	}

	return intendedSum - actualSum
}

File: Tatsuki_Yabe/Q8/Typescript/gauss.ts
function missingNumber2(nums: number[]): number {
  var n = nums.length;
  var intendedSum = (n * (n + 1)) / 2;
  var actualSum = 0;

  for (const num of nums) {
    actualSum += num;
  }

  return intendedSum - actualSum;
}
